Code/src/main/java/nl/utwente/viskell/ui/components/DrawWire.java

```diff
@@ -0,0 +1,124 @@
+package nl.utwente.viskell.ui.components;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.geometry.Point2D;
+import javafx.scene.shape.CubicCurve;
+import javafx.scene.transform.Transform;
+import nl.utwente.ewi.caes.tactilefx.control.TactilePane;
+import nl.utwente.viskell.ui.ComponentLoader;
+import nl.utwente.viskell.ui.CustomUIPane;
+
+/**
+ * A DrawWire is a helper class to represent a new incomplete connection is the process of being drawn. 
```

Comment:
Calling DrawWire a helper class makes it sound like a utility class, which it clearly isn't (utility classes don't usually have constructors).